### ISSUE TITLE
Overnight recordings die.

### ISSUE THREAD

**Issue by JakeCigar** [sahilchaddha/node-rtsp-recorder#4]
You need to be a little more aggressive creating directories. At least on my Mac.

In recordStream, you need to add

```javascript
    fh.createDirIfNotExists(this.getTodayPath())
```
before
```javascript
    fh.createDirIfNotExists(folderPath)
```

**Comment by iwonder-1618** [comment]
@JakeCigar I agree, looks like nobody tested the plugin with overnight recording. I will create a PR with a potential fix soon.

**Comment by JakeCigar** [comment]
I’ve been running several days with the added call to fh.createDirIfNotExists. You might want to add some recursive mkdir type code to replace fh.createDirIfNotExists.

**Comment by Niranjanvelu** [comment]
Could you please share the code for this?
